bump-doozer

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+git diff --quiet
+
+old_version=$(cat doozerlib/VERSION)
+echo $old_version
+
+new_version=$( awk 'BEGIN{OFS=FS=".";}{$NF=$NF+1; print}' <<<"$old_version")
+
+echo "$new_version" >doozerlib/VERSION
+
+git add doozerlib/VERSION
+
+git commit -evm "Prepare for release v$new_version
+
+$(git log --oneline --no-merges v${old_version}..)"
```

Comment:
what about:
```suggestion
$(git log --oneline --no-merges v${old_version}..)"

git tag "v$new_version"
if [[ "$1" != "" ]]; then
    git push "$1" && git push "$1" "v$new_version"
done
```

Reply:
Tagging presumes that the proposal to cut a release is accepted and merged. Prefer to not touch that yet, and keep this a manual action :grimacing: